budgetkey_data_pipelines/pipelines/procurement/tenders/exemptions/add_publisher_urls_resource.py

```diff
@@ -15,11 +17,24 @@
              45: []}
 
 def resource_filter(resource_data, parameters):
+    max_pages = os.environ.get("EXEMPTION_PUBLISHERS_MAX_PAGES", "")
```

Comment:
e.g DEBUG_EXEMPTION_PUBLISHER_MAX_PAGES or TEST_EXEMPTION_PUBLISHER_MAX_PAGES so that these are not confused to be part of the official API.